examples/angular/src/app/buttonWrapper/button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA, EventEmitter, input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.css'],
+})
+export class ButtonComponent {
+  buttonVariation = input<string>('solid');
+  additionalClasses = input<string>('');
+  @Output() buttonClickEvent = new EventEmitter<string>();
+
+  handleClick() {
+    this.buttonClickEvent.emit('warn');
+  }
```

Comment:
Does this event have anything to do with the underlying `ds-click` event?

Reply:
Yes it does. I realize looking at this again that maybe I don't need the double event emission. Basically the ds-click event calls this function which then passes a value to the listener on the wrapping component, which then passes the event value to the parent component. This is how I came up with passing events from the web component to the wrapper to the parent component.

Buuuut maybe I can just listen on the wrapper for the ds-click event to bubble and catch it there? I think whether that works might depend on the presence of the shadow DOM, but would be good to test.

Reply:
The `ds-click` listener is in the button.component.html wrapper. It then emits the custom event here so that the changes from within this component make it up to the app.component. 

The listener for that custom event emission is in the app.component.html on line 14:
```
...
<app-button (buttonClickEvent)="setToWarning($event)" [buttonVariation]="solidVariation">
...
```

Reply:
Oh sorry, I think I was looking at the wrong template